### ISSUE TITLE
No sound

### ISSUE THREAD

**Issue by maurehur** [NeverWise/plugin.video.dplay#27]
There is no sound when the video is playing!

**Comment by NeverWise** [comment]
Yes, I think the url we use to get the stream url is no longer good but I wait to have confirmations.

**Comment by martusi61** [comment]
Same problem here, with CoreElec 9.2.5, Kodi 18.9 and Dplay updated to version 1.0.8.3.

**Comment by riccardosimo** [comment]
Same here... I thought I was getting deaf 👀🥺😂

**Comment by puppinoo** [comment]
Same here. No sound in Leia 18.9.

Thanks

**Comment by Mat-Ley** [comment]
Hi. I wonder if there is a way to solve it? 

thanks a lot in advance,
Matley

**Comment by puppinoo** [comment]
> Hi. I wonder if there is a way to solve it?
> 
> thanks a lot in advance,
> Matley
Using this for now. Older stuff is not free but newest is:
[https://forum.kodi.tv/showthread.php?tid=340055](https://forum.kodi.tv/showthread.php?tid=340055)
